src/app/transactions/bar-chart/charts.scss

```diff
@@ -0,0 +1,9 @@
+.bar-chart {
```

Comment:
this style should be a common style file for bar chart and pie chart, should have name smth like `.chart` and should be inside `app/styles`. See the example of such file in vanilla